modules/organization-policy/README.md

```diff
@@ -0,0 +1,167 @@
+  
+  policies = {
+    "folders/1234567890" = {
+      # enforce boolean policy with no conditions
+      "iam.disableServiceAccountKeyUpload" = {
+        rules = [
+          {
+            enforce = true
+          }
+        ]
+      },
+      # Deny All for compute.vmCanIpForward policy
+      "compute.vmCanIpForward" = {
+        inherit_from_parent = false
+        rules = [
+          deny = [] # stands for deny_all
+        ]
+      }
+    },
+    "organizations/1234567890" = {
+      # allow only internal ingress when match condition env=prod
+      "run.allowedIngress" = {
+        rules = [
+          {
+            allow = ["internal"]
+            condition = {
+              description= "allow ingress"
+              expression = "resource.matchTag('123456789/environment', 'prod')"
+              title = "allow-for-prod-org"
+            }
+          }
+        ]
+      }
+    }
+  } 
+}
+# tftest skip
+```
+
+## Org Policy definition format and structure
+
+### Structure of `policies` variable
+
+```hcl
+policies = {
+  "parent_id" = { # parent id in format projects/project-id, folders/1234567890 or organizations/1234567890.
+    "policy_name" = { # policy constraint id, for example compute.vmExternalIpAccess.
+      inherit_from_parent = true|false # (Optional) Only for list constraints. Determines the inheritance behavior for this policy.
+      reset               = true|false # (Optional) Ignores policies set above this resource and restores the constraint_default enforcement behavior.
+      rules               = [ # Up to 10 PolicyRules are allowed.
+        {
+          allow = ["value1", "value2"] # (Optional) Only for list constraints. Stands for `allow_all` if set to empty list `[]` or to `values.allowed_values` if set to a list of values
+          denyl = ["value3", "value4"] # (Optional) Only for list constraints. Stands for `deny_all` if set to empty list `[]` or to `values.denied_values` if set to a list of values
+          enforce   = true|false       # (Optional) Only for boolean constraints. If true, then the Policy is enforced.
+          condition = {                # (Optional) A condition which determines whether this rule is used in the evaluation of the policy.
+            description = "Condition description" # (Optional)
+            expression  = "Condition expression"  # (Optional) For example "resource.matchTag('123456789/environment', 'prod')".
+            location    = "policy-error.log"      # (Optional) String indicating the location of the expression for error reporting.
+            title       = "condition-title"       # (Optional)
+          }
+        }
+      ]
+    }
+  }
+}
+# tftest skip
+```
+
+### Structure of configuration provided in a yaml file/s
+
+Configuration should be placed in a set of yaml files in the config directory. Policy entry structure as follows:
+
+```yaml
+parent_id: # parent id in format projects/project-id, folders/1234567890 or organizations/1234567890.
```

Comment:
I would put the parent id in the file name, so that one file maps to a single parent only. Managing multiple parents in a single file makes it harder to manage this.

Just my personal PoV so feel free to ignore. :)